### PR TITLE
Use SplashScreen API

### LINKED ISSUE #3719: Migrate to SplashScreen API (Android 12)
See https://developer.android.com/guide/topics/ui/splash-screen/migrate

### Test Notes
test cold and warm starts with dark theme, light theme, whitelabel on 
* [ ] Android 7
* [ ] Android 10
* [ ] Android 12 or 13

### PR DESCRIPTION
**Pull request by wrdhub**
Use SplashScreen API

close #3719

### CODE REVIEW

File: app-android/app/build.gradle [hunk 1]
```diff
@@ -113,10 +113,11 @@ dependencies {
 	//noinspection GradleDependency
 	implementation 'commons-io:commons-io:2.5'
 
-	implementation 'androidx.core:core-ktx:1.8.0'
+	implementation "androidx.core:core-ktx:1.8.0"
 	implementation "androidx.activity:activity-ktx:$activity_version"
 	implementation "androidx.browser:browser:1.3.0"
 	implementation "androidx.biometric:biometric:1.1.0"
+	implementation "androidx.core:core-splashscreen:1.0.0-beta02"
```
Comment (charlag) [inline app-android/app/build.gradle:120]: there's a stable release?
https://developer.android.com/jetpack/androidx/releases/core#core-splashscreen-1.0.0

**Review by charlag — COMMENTED** [review]
I didn't try it out but from a quick glance it looks good!

**Comment by ganthern** [comment]
looks fine now, what versions do we need to test this on? I'd guess something like 7 as the lowest we support, 10 as a pre-splashscreen, and 12 for a newer one?

**Comment by wrdhub** [comment]
Yes, that should be good.

**Review by ganthern — APPROVED** [review]

**Pull request merged**